### ISSUE TITLE
Abandoned Land to Starting Homestead in 6 Months

### ISSUE THREAD

**Issue by nelsonic** [dwyl/phase-three#121]
Abandoned Land to Starting Homestead in 6 Months: https://youtu.be/yORlix6CV60
<img width="766" alt="image" src="https://user-images.githubusercontent.com/194400/188831141-f51d7406-76be-49d1-9244-2ed6faf65878.png">

They are based in PT, but the location is not disclosed. Makes sense to not get randos appearing at their doorstep ... 🙄

**Comment by nelsonic** [comment]
<img width="766" alt="image" src="https://user-images.githubusercontent.com/194400/188831897-ca854bac-249c-4fb9-acc5-8055bb31181f.png">

WASPA: https://goodlifepermaculture.com.au/tag/waspa/

**Comment by nelsonic** [comment]
Using a mini excavator to do the "hard work" when planting trees: 

<img width="769" alt="image" src="https://user-images.githubusercontent.com/194400/188832806-6ebe3a81-cf32-4a4b-9d99-0f23f4cbd026.png">

ref: #113

**Comment by nelsonic** [comment]
Really stoked that there are ***`electric`*** Excavators & Loaders now! 🎉 
https://www.alibaba.com/product-detail/EVERUN-New-Material-ERE12E-1100kg-s355_1601208388385.html
<img width="1599" alt="image" src="https://github.com/user-attachments/assets/80e9c0cf-4222-4992-bc74-e10bd1294450">

https://www.alibaba.com/product-detail/Everun-CE-Certified-EREL04-0-4t_1601032996952.html
<img width="1599" alt="image" src="https://github.com/user-attachments/assets/d6f472c5-bcb5-4b41-9a07-29202ad0b10c">

Seller: https://everun.en.alibaba.com/productgrouplist-940915306/EVERUN_Electric_Wheel_Loader.html

Ref: https://www.nesherequipment.com/about they are re-branding the Chinese machines.
<img width="1188" alt="image" src="https://github.com/user-attachments/assets/05427a12-980f-4370-8082-c6481be582a8">

See: https://www.youtube.com/watch?v=u4nBoWp9Rzo

**Comment by nelsonic** [comment]
https://electrek.co/2025/02/03/i-bought-a-container-full-of-chinese-electric-excavators-heres-what-showed-up/

**Comment by carolrmc** [comment]
This is truly an amazing story of innovation and persistence!
Looks like the products meet the standards too. Would they be available
here?

**Comment by nelsonic** [comment]
Yes, there will be shipping and import duties to be paid on top of the list price.
But I think it's worth it to have a (heavy) tool we can use to dig trenches/holes, move materials, etc.
_without_ burning any more fossil fuels (beyond manufacturing & transporting it). 
One day (_much sooner than most people realise_) fossil fuels will be _finished_ ... ⏳
